config/crd/cluster.open-cluster-management.io_hypershiftdeployments.yaml

```diff
@@ -7049,8 +7044,22 @@ spec:
                   - spec
                   type: object
                 type: array
+              override:
+                description: InfrastructureOverride allows support for special cases   InfraOverrideDestroy
+                  = "ORPHAN"   InfraConfigureOnly = "INFRA-ONLY"   InfraConfigureWithManifest
```

Comment:
Doesn't need to be changed for this merge, but we should take the term `Infra` out of the orphan and ManifestWork.